vysledek/index.html

```diff
@@ -0,0 +1,199 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Cestování po Islandu</title>
+    <link rel="stylesheet" href="style.css">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Karla&family=Montserrat&family=Raleway:wght@300;400;500;600;700&family=Roboto:wght@300;400;700&family=Source+Sans+Pro&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <div class="container">
+
+
+        <header>
+            <h1 class="title">Island</h1>
+
+            <p>Když se řekne Island, spoustě lidem se jako první vybaví sníh a led. Sama jsem tam strávila téměř rok života a zem pokrytou sněhem a ledem jsem zažila jen jediný den. Jednou věcí si ale jistá jsem: že návštěva ostrova pod polárním kruhem je pro každého tak silným zážitkem, že se sem bude chtít vrátit.</p>
+
+            <p class="signature"><i>Jana Zcela Vymyšlená</i></p>
+        </header>
+
+        <div class="leading">
+            <div class="together">
+                <nav>
+                    <ul>
+                        <li><a href="#uvod">Úvod</a></li>
+                        <li><a href="#proc">Proč jet na Island</a></li>
+                        <li><a href="#co">Co musíte vidět</a></li>
+                        <li class="noborder"><a href="#galerie">Fotogalerie</a></li>
```

Comment:
Tady by ani třída být použitá nemusela, vzpomeň, že známe pokročilejší pseudoselektor :last-child https://developer.mozilla.org/en-US/docs/Web/CSS/:last-child, pomocí kterého můžeme border pro poslední položku přepsat 🙂